packages/backend/src/managers/recipes/RecipeManager.ts

```diff
@@ -96,73 +96,70 @@ export class RecipeManager implements Disposable {
     });
   }
 
-  public async buildRecipe(
-    connection: ContainerProviderConnection,
-    recipe: Recipe,
-    model: ModelInfo,
-    labels?: { [key: string]: string },
-  ): Promise<RecipeComponents> {
-    const localFolder = path.join(this.appUserDirectory, recipe.id);
+  public async buildRecipe(options: ApplicationOptions, labels?: { [key: string]: string }): Promise<RecipeComponents> {
```

Comment:
why not including the labels in the options? feel weird to move everything inside ApplicationOptions but the labels?

Reply:
First reason is because labels is not part of the parameters of `requestPullApplication`, and I would like to keep the same options structure everywhere